Feature: add date time to config for DateTime and DateTimePicker

Michael-HEIW opened the pull request:
Allows the config for DateTime to be set in config\forms.php, or defaults to the same current standard

I hope this is acceptable.

Michael-HEIW committed:
feature: add date time to config for DateTime and DateTimePicker

Allows the config for the DateTime to be set, or defaults to the same current standard

I hope this is acceptable.

danharrin commented:
I believe this breaks the TimePicker, we also need formats for `time` and `time_with_seconds`

danharrin commented on packages/forms/config/forms.php:
```diff
@@ -1,6 +1,19 @@
+
+    'date_format' => 'M j, Y',
+    'date_time_format' => 'M j, Y H:i',
+    'date_time_second_format' => 'M j, Y H:i:s',
```
Maybe rename to `date_time_with_seconds_format`

Michael-HEIW committed:
feature: add date time to config for DateTime, DateTimePicker and TimePicker

- revised to `date_time_with_seconds_format`
- added config for:
    - 'time_format',
    - 'time_with_seconds_format',

I'm not sure if this feels right, please let me know your feeling.

danharrin committed:
wip
